frontend/jupyter/src/app/pages/form/form-new/form-new.component.html

```diff
@@ -29,41 +29,67 @@
         [nsMetadata]="nsMetadata"
       ></app-form-image>
 
-      <app-form-cpu-ram
-        [parentForm]="formCtrl"
-        [readonlyCPU]="config?.cpu?.readOnly"
-        [readonlyMemory]="config?.memory?.readOnly"
-        [cpuLimitFactor]="config?.cpu?.limitFactor"
-        [memoryLimitFactor]="config?.memory?.limitFactor"
-        [readonlySpecs]="readonlySpecs"
-        [nsMetadata]="nsMetadata"
-      ></app-form-cpu-ram>
-
-      <app-form-gpus
-        [parentForm]="formCtrl"
-        [vendors]="config?.gpus?.value.vendors"
-        (gpuValueEvent)="checkGPU($event)"
-      ></app-form-gpus>
-
-      <app-form-workspace-volume
-        *ngIf="formCtrl.get('workspace')"
-        [readonly]="config?.workspaceVolume?.readOnly"
-        [volGroup]="formCtrl.get('workspace')"
-        [externalName]="formCtrl.get('name').value"
-      >
-      </app-form-workspace-volume>
-
-      <app-form-data-volumes
-        [volsArray]="formCtrl.get('datavols')"
-        [readonly]="config?.dataVolumes?.readOnly"
-        [externalName]="formCtrl.get('name').value"
-      >
-      </app-form-data-volumes>
+      <lib-form-section title="Protected B">
```

Comment:
I feel like we could probably convert this into its own component (like with what you did with the form-image-custom)